### ISSUE TITLE
Structure of tutorials and examples

### ISSUE THREAD

**Issue by thomasrobiglio** [brainets/hoi#32]
Following the [diataxis](https://diataxis.fr) approach to documentation, I propose the following organization of tutorials and examples.

**EXPLANATION** -> theoretical explanations as already present [here](https://brainets.github.io/hoi/overview/ovw_theory.html) + [references](https://brainets.github.io/hoi/overview/ovw_refs.html)
(might require checking of the equations + adding some figures)

**REFERENCE** -> API documentation

**TUTORIALS**
The goal of this part is to guide the user through the structure of the important functions in the toolbox. 
- [ ]  "Core information theoretical measures"
Here the goal is to show how to compute entropy and mutual information. I think a simple case with entropy/MI of univariate Gaussians is ok.
- [ ] "Higher-order information theoretical measures"
Here we show for a single metric (O-info?) how the `hoi.metrics` class works. Something very similar to the [existing example with RSI](https://brainets.github.io/hoi/auto_examples/metrics/plot_rsi.html#sphx-glr-auto-examples-metrics-plot-rsi-py).
Create a simple data, define the model, fit it, print the best multiplets, and plot the results
- [ ] "Simulation of HOI"

**HOW-TO GUIDES** -> Examples
This consists of a gallery of examples (one per metric?) showing the computation of the given metric on simple data (similar to what is done in the corresponding tutorial but with shorter descriptions).
- [ ] Oinfo
- [ ] InfoTopo
- [ ] TC
- [ ] DTC
- [ ] Sinfo
- [ ] GradientOinfo
- [ ] RSI
- [ ] RedundancyMMI
- [ ] SynergyMMI
- [ ] InfoTot

**Comment by Mattehub** [comment]
@EtienneCmb, I am working on examples and tutorials, it is ok to have one example per metric? For sure there will be some redundancy and repetition: many times the application of different metrics is very similar. But at the same time, there are differences in all the metrics, so if you agree I will add one example per metric.

**Comment by EtienneCmb** [comment]
@Mattehub I agree that the code is going to be highly redundant. Maybe a comparison between metrics might be better. I planned a list of examples, I'm working on this right now. If you can, it might be better to clean your function for simulating HOI using gaussians.

**Comment by Mattehub** [comment]
OK, perfect! Thanks for telling me. So I go to clean the gaussian simulation functions
